frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/type.ts

```diff
@@ -8,6 +8,7 @@ export type Data = {
   highlightedHandles: string[]
   sourceColumnName: string | undefined
   targetColumnCardinalities?: Record<string, Cardinality> | undefined
+  onClick?: (tableName: string) => void
```

Comment:
This does not appear to have been used from TableNode, was it necessary?

Reply:
Oh,sorry, you are right.
I've removed the unnecessary props🙏. https://github.com/liam-hq/liam/pull/284/commits/a518be68e426f6ad81da6cd613249c18448c2738